lib/runner/folder-walk.js

```diff
@@ -39,6 +39,7 @@ class Walker {
     this.settings = lodashClone(settings, true);
     this.argv = argv;
     this.usingMocha = this.settings.test_runner && this.settings.test_runner.type === 'mocha';
+    this.usingCucumber =  this.settings.test_runner && this.settings.test_runner.type === 'cucumber';
```

Comment:
Since we have upgraded to ECMAScript 2020, we can simplify the code snippet as follows:
 
```suggestion
    this.usingCucumber = this.settings.test_runner?.type === 'cucumber';
```